packages/@vue/cli-service/types/ProjectOptions.d.ts

```diff
@@ -23,37 +23,120 @@ interface ExtractOptions {
 }
 
 interface CSSOptions {
+  /**
+   * Default: `true`
+   * By default, only files that ends in `*.module.[ext]` are treated as CSS modules
+   */
   requireModuleExtension?: boolean;
+  /**
+   * Default: `true`
+   * Whether to extract CSS in your components into a standalone CSS files (instead of inlined in JavaScript and injected dynamically)
+   */
   extract?: boolean | ExtractOptions;
+  /**
+   * Default: `false`
+   * Whether to enable source maps for CSS. Setting this to `true` may affect build performance
+   */
   sourceMap?: boolean;
+  /**
+   * Default: `{}`
+   * Pass options to CSS-related loaders
+   */
   loaderOptions?: LoaderOptions;
 }
 
 interface ProjectOptions {
+  /**
+   * Default: `'/'`
```

Comment:
```suggestion
   * Default: `'/'`
   *
```

Reply:
It seems adding a new line after all `Default`, the editor can give more friendly  parameter info.
![image](https://user-images.githubusercontent.com/13298807/97826468-9ad63d00-1cfc-11eb-8690-274f873bfff6.png)

If no new line, 
![image](https://user-images.githubusercontent.com/13298807/97826518-b6d9de80-1cfc-11eb-9a57-197501fbd237.png)

test by windows vscode